### PR TITLE
PH-124_add_release_guide

### PR DESCRIPTION
**Pull request by TheKeymaster**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,15 @@ In order to start an export:
...
+
+## Deployment & Release
+
+1. Ensure all necessary changes have been merged into the `develop` branch.
+1. Merge all changes from `develop` into `main`.
+1. Create a new release on GitHub. As branch use `develop`. This will **automatically** create
+a tag for this repository.
+1. Ensure that the **build passed successfully** and bump the version of the REST exporter in
+the importer by running `composer require findologic/plentymarkets-rest-export:^1.35`.
+   * Ensure that you specify `^` and **only** two versions e.g. `^1.35` or `^2.12`.
```
Comment (dblaichinger) [inline README.md:19]: "two versions" might be confusing, let's call it Major and Minor version.

File: README.md [hunk 2]
```diff
@@ -6,4 +6,15 @@ In order to start an export:
...
+1. From here on follow the usual importer deployment procedure.
```
Comment (dblaichinger) [inline README.md:20]: It would be convenient if you add a link to the deploy instructions at http://build.findologic.com/infrastructure/backend#deployment

**Review by dblaichinger — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (TheKeymaster) [reply README.md:20]: This is a public repository, why should we share internal links?

Comment (dblaichinger) [reply README.md:20]: Right, if it's public we should avoid it

**Pull request merged**